skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg

```diff
@@ -71,10 +71,19 @@
 - And at what time?
 - Got it, and when do you want it to start?
 
+# AskForTimePrompt(noSuggestion)
+- IF: ${ noSuggestion == true }
+    - I can't suggest a time to meet, probably because there is no overlap in working hours. What day did you want to meet?
+    - There's aren't any relevant suggestions, probably there is no overlap in working hours. What day did you have in mind?
+    - I couldn't find relevant suggestions that works for all attendees. What day did you want to meet?
```

Comment:
I couldn't find relevant suggestions that work for all attendees. What day did you want to meet?

Reply:
fixed